hack/validate/golangci-lint.yml

```diff
@@ -39,3 +39,9 @@ issues:
     - text: "G202: SQL string concatenation"
       linters:
         - gosec
+    - text: "SA1019: h.Xattrs is deprecated: Use PAXRecords instead"
```

Comment:
this might be a legit warning... I started to take a look at it and got carried away :(

Anyway, let's address it as a followup.

Reply:
Yes, I think all (or most) of the ones I'm ignoring in this file are legit, and things that should be addressed. I want to create tracking issues for those.

😂 I also started working on the `PAXRecords` one, but wasn't sure if my approach was right and/or if it would cause backward-incompatibility issues if we switched; so I removed those changes, and thought it was better left for a separate PR (and discussion)